src/binder/binder_context.cpp

```diff
@@ -18,9 +18,10 @@
 namespace peloton {
 namespace binder {
 
-void BinderContext::AddTable(const parser::TableRef* table_ref) {
+void BinderContext::AddTable(const parser::TableRef* table_ref,
+                             concurrency::Transaction* consistentTxn) {
   storage::DataTable* table = catalog::Catalog::GetInstance()->GetTableWithName(
-      table_ref->GetDatabaseName(), table_ref->GetTableName());
+      table_ref->GetDatabaseName(), table_ref->GetTableName(), consistentTxn);
```

Comment:
I don't think it should be called "consistent" anything. It doesn't make sense. It seems to imply that other txns are not consistent. It's confusing.

Change it to be just "txn"